src/app/onion/learning-object-builder/learning-object-builder.routing.ts

```diff
@@ -1,4 +1,5 @@
-import { ModuleWithProviders } from '@angular/core';
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why is it angry at Angular?

Reply:
It's just clark's very strict British nanny

Reply:
Haha